speechbrain/core.py

```diff
@@ -542,9 +585,9 @@ def __init__(  # noqa: C901
             "compile_mode": "reduce-overhead",
             "compile_using_fullgraph": False,
             "compile_using_dynamic_shape_tracing": False,
-            "auto_mix_prec": False,
-            "bfloat16_mix_prec": False,
+            "precision": "fp32",
             "max_grad_norm": 5.0,
+            "skip_nonfinite_grads": False,
             "nonfinite_patience": 3,
             "noprogressbar": False,
             "ckpt_interval_minutes": 0,
```

Comment:
I'd like to see the run opts become a dataclass at some point, not sure if this PR is the right time for it.

Reply:
Maybe even put in a separate file.